src/CRM/CivixBundle/Resources/views/Code/module.php.php

```diff
@@ -3,7 +3,11 @@
 $_namespace = preg_replace(':/:', '_', $namespace);
 ?>
 
+declare(strict_types = 1);
+
+// phpcs:disable PSR1.Files.SideEffects
 require_once '<?php echo $mainFile ?>.civix.php';
+// phpcs:enable
```

Comment:
What's the purpose of these phpcs comments? `civix` already writes code that passes all the checks run by the `civilint` command. Anyone using `civilint` should be fine without this?

Reply:
The phpcs configuration in [systopia/civicrm-extension-template](https://github.com/systopia/civicrm-extension-template/blob/main/phpcs.xml.dist.template) is stricter than civilint and adds this sniff. Since each extension uses this pattern (while not encouraged elsewhere), we find ourselves adding these comments in every extension after civix built the file.

So technically, you're right, but I'd say these comments don't hurt … :shrug: